server/services/auth/password.go

```diff
@@ -23,9 +23,11 @@ const (
 	InvalidSymbolPassword    = "symbol"
 )
 
+var PasswordHashStrength = 10
```

Comment:
@wiggin77 and @jespino - added this variable so that unit tests can reduce it (making register and login calls go from around 0.8 to 1 second to near instantaneous). We should be careful that this variable isn't accidentally set during production, and open to other ways to refactor this.

Reply:
There is the `FB_UNIT_TESTING` environment variable that is used in other places to change config for unit tests.

Reply:
I did consider that, but wasn't sure if that was better or worse than setting the variable directly. There's a (small) risk that environment variables (unintentional or malicious) could affect a production environment. Do you have a strong preference one way or another?

Reply:
0/5